doc/antora/modules/reference/pages/man/index.adoc

```diff
@@ -1,4 +1,4 @@
-= Man pages
+= Man Pages
```

Comment:
? Why are some headings changed from upper case to lower case, and others change the opposite way?

I had thought that headings should generally use upper case for the words?